README.md

```diff
@@ -56,6 +56,7 @@ environments.
 
 | Ceph CSI Version | Container Orchestrator Name | Version Tested     |
 | -----------------| --------------------------- | -------------------|
+| v3.11.1          | Kubernetes                  | v1.27, v1.28, v1.29, v1.30|
 | v3.11.0          | Kubernetes                  | v1.26, v1.27, v1.28, v1.29|
```

Comment:
We don't have 3.11.1 yet. Let's add the version when we do the release.

Reply:
I added that so we won't forget to update it. Do you prefer a note in `PendingReleaseNotes.md` instead?

Reply:
We can just check while doing the release and update in Readme IMO

Reply:
Ok, removed the README.md change.